packages/documentation/src/stories/health/icons/icons.blocks.tsx

```diff
@@ -0,0 +1,104 @@
+// icons.blocks.tsx
+import React from 'react';
+import reportData from '../../../../node_modules/@swisspost/design-system-icons/public/report.json';
```

Comment:
```suggestion
import reportData from '@root/node_modules/@swisspost/design-system-icons/public/report.json';
```

Not 100% sure if this works!